<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimAnnealingClass</class>
 <widget class="QMainWindow" name="SimAnnealingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimAnnealing</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_main">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 9pt &quot;Noto Sans CJK SC Light&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Simulated annealing&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="view_anneal">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>90</y>
      <width>300</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="view_greedy">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>300</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_greedy">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 9pt &quot;Noto Sans CJK SC Light&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;greedy&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_anneal">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>300</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 9pt &quot;Noto Sans CJK SC Light&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;anneal&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="input_num">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>340</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC Light</family>
      <pointsize>9</pointsize>
      <weight>3</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 9pt &quot;Noto Sans CJK SC Light&quot;;</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans CJK SC Light'; font-size:9pt; font-weight:24; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;45&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_num">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>380</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 9pt &quot;Noto Sans CJK SC Light&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;num&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="button_start">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 9pt &quot;Noto Sans CJK SC Light&quot;;</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_heattext">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>370</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC Light</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Heat:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_heatvalue">
    <property name="geometry">
     <rect>
      <x>571</x>
      <y>371</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK SC DemiLight</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NULL</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="simannealing.qrc"/>
 </resources>
 <connections/>
</ui>
